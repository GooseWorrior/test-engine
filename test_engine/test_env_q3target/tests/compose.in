*|*

a
q
*||*

a
q
*|*|*

a
q
**|**

a
q
(a|b)(c|d)|e
ab
ac
bc
bd
e
abcde
ace
bde
c*d|a
cd
ccd
a
d
ca
cca
q
(a*|b)*

a
aa
ab
abb
abaabaaa
bb
q

