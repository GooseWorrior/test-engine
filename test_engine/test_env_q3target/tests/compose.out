Disjunction (Star (),Star ())
true
false
Disjunction (Star (),Disjunction (,Star ()))
true
false
Disjunction (Star (),Disjunction (Star (),Star ()))
true
false
Disjunction (Star (Star ()),Star (Star ()))
true
false
Disjunction (Concatenation (Disjunction (a,b),Disjunction (c,d)),e)
false
true
true
true
true
false
false
false
false
false
false
false
false
false
false
Star (Disjunction (Star (a),b))
true
true
true
true
true
true
true

